scripts/mover.py

```diff
@@ -83,15 +94,54 @@ def stop_start_torrents(torrent_list, pause=True):
 
     torrents = filter_torrents(torrent_list, timeoffset_from.timestamp(), timeoffset_to.timestamp(), args.cache_mount)
 
-    # Pause Torrents
     print(f"Pausing [{len(torrents)}] torrents from {args.days_from} - {args.days_to} days ago")
-    stop_start_torrents(torrents, True)
+    stop_start_torrents(torrent_list)
+
+    file_paths = set()
+    link_paths = set()
+    dir_paths = set()
+
+    for torrent in torrents:
+        content_path = cache_path(args.cache_mount, torrent.content_path)
+
+        if os.path.isdir(content_path):
+            # If file_path is a directory, include all files within it
+            for root, dirs, files in os.walk(content_path, topdown=False):
+                for file in files:
+                    file_path = os.path.join(root, file)
+                    file_paths.add(os.path.join(root, file))
+
+                for dir in dirs:
+                    directory_path = os.path.join(root, dir)
+                    dir_paths.add(directory_path)
+            dir_paths.add(content_path)
+        else:
+            file_paths.add(content_path)
+
+    for file_path in file_paths:
+        for link in find_hardlinks(content_path, args.cache_mount):
+            if link not in file_paths:
+                link_paths.add(link)
+
     time.sleep(10)
-    # Start mover
-    print("Starting Mover")
```

Comment:
Why remove this line?

Reply:
as we have it bellow, per each category: files, links, dirs